llvm/lib/Support/KnownBits.cpp

```diff
@@ -774,6 +774,39 @@ KnownBits KnownBits::usub_sat(const KnownBits &LHS, const KnownBits &RHS) {
   return computeForSatAddSub(/*Add*/ false, /*Signed*/ false, LHS, RHS);
 }
 
+KnownBits KnownBits::avgCompute(const KnownBits &LHS, const KnownBits &RHS,
+                                bool IsCeil, bool IsSigned) {
+  KnownBits Known = LHS;
```

Comment:
It might be neater to pass LHS and RHS in by value, instead of by const ref, so that you don't need to copy them here.